### ISSUE TITLE
decide on a consistent style guide

### ISSUE THREAD

**Issue by rafd** [braidchat/meta#1]
e.g. parameters on the same line as function line or line below
destructure map params (I like doing `(defn f [{:keys [a b c]}] …)` so I can see what the map keys should be when I do `(doc f)`

**Comment by pkobrien** [comment]
think about why you do this
think about the value to you as author and others as readers
think about how any of that can apply to the things being communicated via chat
think about how chat communication is NOT the same as code communication (emotional content)
think about the true underlying value(s) of emoji and reactji

**Comment by rafd** [comment]
fine with those; perhaps you want to write a kibbitz-style linter for all our projects? :smile:

**Comment by rafd** [comment]
or maybe actually use kibit (https://github.com/jonase/kibit) and add new patterns

**Comment by rafd** [comment]
Writing our own seems more fun :p

**Comment by pkobrien** [comment]
Kibit looks cool. I'm not sure how well that would work with analyzing chat messages, Might be worth trying. I was more just thinking about how efficiently I can communicate using Braid. And emoji and reactji are super efficient (the entirety of the "message" encapsulated in a single icon added with a single click or oft-practiced key combinations) at expressing emotions.

**Comment by jamesnvc** [comment]
Oh, think this issue was specifically referring to a style guide for the code. A style guide for chatting is a really good idea though!

**Comment by pkobrien** [comment]
Computer code is devoid of emotion. Human beings are not. When communication is devoid of emotion the human is left to supply that piece themselves. That's why the history of IRC and such are littered with hurt feelings and flame wars. Braid will benefit by taking this on front a center. And a "style guide" that helps users express emotions using Braid will have the same benefit as a style guide for code.

**Comment by jamesnvc** [comment]
Agreed! I created an issue a user style guide on Asana, which I think should get mirrored here

**Comment by jamesnvc** [comment]
See https://github.com/braidchat/meta/issues/80
